test/plugins/validation/oas3/parameters.js

```diff
@@ -224,7 +284,219 @@ describe('validation plugin - semantic - parameters - oas3', function() {
+    const config = {
+      parameters: {
+        no_in_property: 'error',
+        invalid_in_property: 'error',
+        missing_schema_or_content: 'error',
+        has_schema_and_content: 'error'
+      },
+      schemas: {
+        json_or_param_binary_string: 'warning'
+      }
+    };
+
+    const spec = {
+      components: {
+        parameters: {
+          BadParam: {
+            in: 'path',
+            name: 'path_param',
+            description: 'a parameter',
+            content: {
+              'text/plain': {
+                type: 'string'
+              },
+              'application/json': {
+                schema: {
+                  type: 'string',
+                  format: 'binary'
+                }
+              }
+            }
+          }
+        }
+      }
+    };
+
+    const res = validate({ jsSpec: spec }, config);
+    expect(res.warnings.length).toEqual(1);
+    expect(res.warnings[0].path).toEqual([
+      'components',
+      'parameters',
+      'BadParam',
+      'content',
+      'application/json',
+      'schema'
+    ]);
+    expect(res.warnings[0].message).toEqual(
+      'Parameters should not contain binary (type: string, format: binary) values.'
+    );
+  });
+
+  it('should complain multiple times when multiple parameters use schema type: string, format: binary', function() {
+    const config = {
+      parameters: {
+        no_in_property: 'error',
+        invalid_in_property: 'error',
+        missing_schema_or_content: 'error',
+        has_schema_and_content: 'error'
+      },
+      schemas: {
+        json_or_param_binary_string: 'warning'
+      }
+    };
+
+    const spec = {
+      components: {
+        parameters: {
+          BadParam1: {
+            schema: {
+              type: 'string',
+              format: 'binary'
+            }
+          },
+          BadParam2: {
+            schema: {
+              type: 'string',
+              format: 'binary'
+            }
+          }
+        }
+      }
+    };
+
+    const res = validate({ jsSpec: spec }, config);
+    expect(res.warnings.length).toEqual(2);
+    expect(res.warnings[0].path).toEqual([
+      'components',
+      'parameters',
+      'BadParam1',
+      'schema'
+    ]);
+    expect(res.warnings[0].message).toEqual(
+      'Parameters should not contain binary (type: string, format: binary) values.'
+    );
+  });
+
+  it('should not complain when the schema field is empty', function() {
+    const config = {
+      parameters: {
+        no_in_property: 'error',
+        invalid_in_property: 'error',
+        missing_schema_or_content: 'error',
+        has_schema_and_content: 'error'
+      },
+      schemas: {
+        json_or_param_binary_string: 'warning'
+      }
+    };
+
+    const spec = {
+      components: {
+        parameters: {
+          GoodParam: {
+            schema: {}
```

Comment:
I would actually recommend making this test about the schema field not even being there. That's often a good thing to verify in JS code. Just make `GoodParam` an empty object.